app/api/routers/health_check_router.py

```diff
@@ -1,17 +1,35 @@
-import logging
-from fastapi import APIRouter
-from api.models.health_check_status import HealthCheckStatus
+from typing import Callable
+from fastapi import APIRouter, Request, Response
+from fastapi.routing import APIRoute
+
+from app.api.models.health_check_status import HealthCheckStatus
+
+from app.logger_config import logger
+
+
+class LogRoute(APIRoute):
+    def get_route_handler(self) -> Callable:
+        original_route_handler = super().get_route_handler()
+
+        async def custom_route_handler(request: Request) -> Response:
+            req = f'Received {request.method} {request.url.path}'

```

Comment:
no need to assign the log string into a variable. 
Directly write it in the logger.info(f'')
same for response as well.

Reply:
Modified